src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -243,9 +243,16 @@
 
 (defn discover
   []
-  (let [featured-communities (rf/sub [:communities/featured-contract-communities])]
-    [rn/view
-     {:style (style/discover-screen-container (colors/theme-colors
-                                               colors/white
-                                               colors/neutral-95))}
-     [discover-screen-content featured-communities]]))
+  [:f>
```

Comment:
I think we run into some performance issues doing this.

I think you have to do
```
(defn f-discover []
(fn []
 (rn/use-effect ...
```

```
(defn discover []
 [:f> f-discover]
```